spec/lib/migrator_spec.rb

```diff
@@ -72,6 +72,17 @@
     end
   end
 
+  describe '.bootstrap' do
+    before(:each) do
+      FileUtils.rm(SeedMigration::Migrator.get_migration_files)
+      Rails::Generators.invoke("seed_migration", ["TestMigrationBefore", 5.days.ago.utc.strftime("%Y%m%d%H%M%S")])
+    end
+
+    it 'runs all migrations' do
+      expect{SeedMigration::Migrator.bootstrap}.to change{SeedMigration::DataMigration.count}.by(1)
```

Comment:
would it be interesting to add an assertion on the value of the version column in the DB after running this?